x-pack/legacy/plugins/code/public/components/main/main.scss

```diff
@@ -171,12 +171,12 @@
   white-space: nowrap;
 }
 
+.codeSearch__suggestion-text-container {
```

Comment:
I'm guessing this code has been around for a while, but we should be using snake case in these instances rather than hyphens: `codeSearch__suggestionTextContainer`

Reply:
Yep, I noticed some inconsistencies with BEM classes here; I'm happy to change that. Should I leave the existing classes for now?

Reply:
7b4f233